LICENSES/LicenseRef-Slint-Community.md

```diff
@@ -1,32 +1,38 @@
-# Slint Royalty-free Desktop and Web Applications License, version 1.1
+Community License, version 1.2
+
+# Slint Royalty-free Desktop, Mobile, and Web Applications License
 
 ## Preamble
 
-Slint is a toolkit that can be used to build user interfaces for applications. Slint (hereafter referred to as "Software") is made available under different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540 Hohen Neuendorf, Germany ("SixtyFPS"). The "Slint Royalty-free Desktop and Web Applications License" is suitable for those who develop desktop or web applications and do not want to use open source components under copyleft licenses.
+Slint is a toolkit that can be used to build user interfaces for applications. Slint (hereafter referred to as "Software") is made available under different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540 Hohen Neuendorf, Germany ("SixtyFPS"). The "Slint Royalty-free Desktop, Mobile, and Web Applications License" is suitable for those who develop desktop, mobile, or web applications and do not want to use open source components under copyleft licenses.
```

Comment:
> The "Slint Royalty-free Desktop, Mobile, and Web Applications License" is suitable for those

This phrase of the license text suggests to me that either the license to continue to be named as it is now, or this sentence should be adjusted.